README.md

```diff
@@ -1187,13 +1190,10 @@ interface AppHistory : EventTarget {
   readonly attribute boolean canGoBack;
   readonly attribute boolean canGoForward;
 
-  Promise<undefined> update(USVString url, optional AppHistoryPushOrUpdateOptions options = {});
-  Promise<undefined> update(optional AppHistoryPushOrUpdateFullOptions options = {}); // one member required: see issue #52
-
-  Promise<undefined> push(USVString url, optional AppHistoryPushOrUpdateOptions options = {});
-  Promise<undefined> push(optional AppHistoryPushOrUpdateFullOptions options = {});
+  Promise<undefined> navigate(USVString url, optional AppHistoryNavigateOptions options = {});
+  Promise<undefined> navigate(optional AppHistoryNavigateOptions options = {}); // one member required: see issue #52
```

Comment:
Related to the comment here: it's probably invalid to just have `appHistory.navigate({replace:true})` right? In that case, one member was provided, but it's still invalid. 

Probably hard to put that into the WebIDL format though.

Reply:
Not something I feel super strongly about, but my intuition would have been that `navigate({})` (or say `go(current.key)`) wouldn’t throw, but also wouldn’t do anything except return a fulfilled promise (or more specifically, one that fulfills at the earliest point at which it could be fulfilled if it had had an effect).

Reply:
Interesting. I'll probably have it throw for now since it's easier to make it un-throw than not, but this is worth keeping in mind...